docker/entrypoint.sh

```diff
@@ -100,6 +102,17 @@ run_manage() {
     exec django-admin "$@"
 }
 
+setup_signing_service() {
```

Comment:
This file is the same used by dev environment https://github.com/ansible/galaxy_ng/pull/1076/files#diff-4f5cabe26761257a4d685a6edc7a43e0fe0f78762f50eeb48530f2bd3b3ee7ca we need to make it in a way it works the same code for both environments, or conditionally select a different function for cloud.

Reply:
The function you wrote should work for the most part in the cloud environments. Just the KEY_ID will still need to be parameterized, instead of requiring galaxy3.
Other than that, a new ENV var would also be required instead of re-using $DEV_SOURCE_PATH. e.g. https://github.com/ansible/galaxy_ng/pull/1111/commits/766e16ab2d8ab53005b648f1364942075ec126aa